src/Authentication/Controllers/RequestSystemUserController.cs

```diff
@@ -91,6 +96,13 @@ public async Task<ActionResult<RequestSystemResponse>> CreateRequest([FromBody]
             SystemId = createRequest.SystemId,
         };
 
+        SystemUser? existing = await _systemUserRepository.GetSystemUserByExternalRequestId(externalRequestId);
```

Comment:
Is there a specific reason why the repository layer i called directly here?

Reply:
yes, probably. Will fix.